cmd/sql-proxy-client/main.go

```diff
@@ -168,11 +168,10 @@ func newLocalCertSource(caPath, certPath, keyPath, remoteAddr string, remotePort
 	if err != nil {
 		return nil, err
 	}
-	cert.Leaf = caCert
```

Comment:
@fatih I'm really unsure what this line was originally for? Why was the leaf set to the CA, that seems incorrect to me?

Reply:
The function `newLocalCertSource` is only used for local testing when you provide the certs yourself (instead of retrieving them from our Certs endpoint). Hence I believe this line was added for testing purposes, and removing them wouldn't affect anything user-facing. I don't know precisely why I added it, though. Because it's safe to remove, this looks good for me 👍🏼